README.md

```diff
@@ -3,36 +3,35 @@
 [![Build Status](https://travis-ci.org/RedHat-Israel/ROSE.svg?branch=master)](https://travis-ci.org/RedHat-Israel/ROSE)
 
 This project is a game that has been developed in order to help teach kids python.
-The students need to code the behavior of the car in order to achieve the best score.
+The students are required to code the behavior of the car to achieve the best score.
 
-Here is video of a race, using drivers coded by students:<br/>
-(Click on it to play the video)
+Here is the link to the sample gameplay video of the race (running code from students):
 
 <a href="http://www.youtube.com/watch?feature=player_embedded&v=BEV-CcqTOnw
 " target="_blank"><img src="rose-video-preview.jpg"
 alt="ROSE Race Car Game" width="860" height="720" border="10" /></a>
 
-In this game, two race cars compete to achieve the most points.
-The race car has to recognize the race track, the obstacles, and the bonus areas;
-calculate the best path to take to avoid the pitfalls; and collect bonus points.
-The cars move autonomously on the screen within the race track game with no interference
-from the students. No joystick or mouse would be used.
+In this game, two race cars compete to obtain the maximum number of points.
+The race car must recognize the racing track, the obstacles on the way and the bonus areas;
+calculate the best path to take to avoid any pitfalls; and collect as many bonus points as possible.
+The cars move autonomously on their track, on screen, in the game without any interference or handling
+from the students. No joystick or mouse needs to be used.
  
-In order to control the car movements, the students needs to implement a 'driver'.
-This code is controlling the car and will decide what will be the next action of the car.
+In order to control the movements of the car, the students needs to implement a 'driver'.
+The code that implements 'driver' controls the car and will decide the next action/movement of the car.
 
-For each type of obstacles there is a different action, and different points.
+For each type of obstacle, there is a different action that must be taken, and different points are alloted for the same.
```

Comment:
You don't have to take the specified action. If you choose the best action, you will get the highest score.